src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,54 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <>
```

Comment:
you can use destructuring here

Reply:
@Jamal-Amoian does the distructurisation as @Alzay007 advised you.